tests/MongoDB.Driver.Tests/RetryableWritesTests.cs

```diff
@@ -43,6 +51,28 @@ public void Insert_with_RetryWrites_true_should_work_whether_retryable_writes_ar
             }
         }
 
+        [SkippableFact]
+        public void Retryable_write_operation_should_throw_custom_exception_on_servers_using_mmapv1()
+        {
+            RequireSupportForRetryableWrites();
+            RequireServer.Check().StorageEngine("mmapv1");
+
+            using (var client = GetClient())
+            using (var session = client.StartSession())
+            {
+                var database = client.GetDatabase(DriverTestConfiguration.DatabaseNamespace.DatabaseName);
+                var collection = database.GetCollection<BsonDocument>(DriverTestConfiguration.CollectionNamespace.CollectionName);
+                var document = new BsonDocument("x", 1);
+                var exception = Record.Exception(() => collection.InsertOne(document));
+
+                exception.Message.Should().Contain(
+                    "This MongoDB deployment does not support retryable writes. " +
+                    "Please add retryWrites=false to your connection string.");
+
+                _output.WriteLine("The cake isn't a lie.");
```

Comment:
I don't understand this line.

Or why _output was added to this class.

Reply:
Oops, sorry, I was using _output to try to track that the test actually ran in Evergreen.

As for the exact line, [this ](https://en.wiktionary.org/wiki/the_cake_is_a_lie#English)is a pretty good summary of the reference to an absolutely amazing game  :D